src/YEthStakerStrategy.sol

```diff
@@ -360,6 +395,16 @@ contract YEthStakerStrategy is
     function availableWithdrawLimit(
         address _owner
     ) public view override returns (uint256) {
+        // review:
+        // you can do:
+        /*
+          uint256 fromDepositFacility;
+          if (address(depositFacility) != address(0)) {
+              (, fromDepositFacility) = depositFacility.available();
+          }
+          return asset.balanceOf(address(this)) + fromDepositFacility + maxSingleWithdraw
```

Comment:
This change is less gas efficient, but we can change it for better readability.

Gas comparison:
```logs
test_profitableReport_withDepositFacility(uint256,uint16) (gas: 6 (0.000%))
test_deposit_withDepositFacility(uint256) (gas: 5 (0.000%))
test_deposit_wethMoreValue_withDepositFacility_fullAmount(uint256,uint8) (gas: 5 (0.001%))
test_profitableReport_withFees_weth_withDepositFacility(uint256,uint16,uint8) (gas: 11 (0.001%))
test_profitableReport_withFees_withDepositFacility(uint256,uint16) (gas: 11 (0.001%))
test_operation(uint256) (gas: 71 (0.008%))
test_reportTrigger(uint256) (gas: 71 (0.009%))
test_deposit_weth_more(uint256) (gas: 71 (0.011%))
test_shutdownCanWithdraw(uint256) (gas: 71 (0.012%))
test_profitableReport_withFees_weth(uint256,uint16) (gas: 142 (0.012%))
test_RevertWhen_withdrawAboveMax() (gas: 72 (0.014%))
test_profitableReport_withFees(uint256,uint16) (gas: 142 (0.017%))
test_deposit_skip(uint256) (gas: 71 (0.017%))
test_deposit_skip_below_wad() (gas: 72 (0.019%))
test_profitableReport(uint256,uint16) (gas: 391 (0.049%))
test_profitableReport_weth(uint256,uint16) (gas: 624 (0.058%))
```